### ISSUE TITLE
Missing parts in BOM

### ISSUE THREAD

**Issue by maximelouet** [vroland/epdiy-hardware#14]
Hi, and thanks a lot for providing all of this!!

I'm trying to get the board printed and assembled through JLCPCB but I get the following error when uploading the gerbers and BoM files:

```
The below parts won't be assembled due to data missing.
J2,J3,J5,J6,J7 designators don't exist in the BOM file.
```

![jlcpcb-error](https://github.com/vroland/epdiy-hardware/assets/8074940/3210265a-9dd8-42c1-b5b6-51fd32164b83)

Indeed, when checking in the BoM file, these 5 items are missing. These parts don't seem to be optional, so I don't really want to continue without them.

It seems they've been removed from the BoM in this commit: https://github.com/vroland/epdiy-hardware/commit/f67d0ec701ee9243f95e018740200eee1eb12c22, I don't know if it was intentional.

Thanks for your help!

**Comment by martinberlin** [comment]
Hi Máxime,
J* symbols in KiCad refer to Jumpers or Sockets which are non parts usually. It could be that are in the POS file and that’s why this alert comes. Try opening with KiCad and check it out. Is a good idea to learn using it if you will send PCBs to fabrication

**Comment by martinberlin** [comment]
Closed #16 since was a duplicate of this.
Will check tomorrow to mark this parts not to be exported in BOM /CPL

**Comment by BackSlasher** [comment]
@martinberlin apologies if it's a stupid question - do you consider int a valid workaround to remove the J parts from the positioning file and submit to the assembling store?
Will I not be missing parts?

**Comment by martinberlin** [comment]
Not really. You should install KiCad and check it out. It’s not essential but recommended if you send stuff to fabrication, then you can check everything, from schematic to PCB.
If you check schematic you will see also what are those parts. Usually J* are jumpers so this are not placeable components just drawn in the Gerbers files
